src/sections/cardholders/cardholder-create-widget.tsx

```diff
@@ -29,6 +31,63 @@ import {
   postApi,
 } from "src/utils/api-helpers";
 
+const validCardholderCountries = (
+  country: string,
+): { name: string; code: string }[] => {
+  if (country == "US") {
+    return [{ name: "United States", code: "US" }];
+  }
+
+  if (country == "GB") {
+    return [{ name: "United Kingdom", code: "GB" }];
+  }
+
+  return [
+    { name: "Austria", code: "AT" },
+    { name: "Belgium", code: "BE" },
+    { name: "Croatia", code: "HR" },
+    { name: "Cyprus", code: "CY" },
+    { name: "Estonia", code: "EE" },
+    { name: "Finland", code: "FI" },
+    { name: "France", code: "FR" },
+    { name: "Germany", code: "DE" },
+    { name: "Greece", code: "GR" },
+    { name: "Ireland", code: "IE" },
+    { name: "Italy", code: "IT" },
+    { name: "Latvia", code: "LV" },
+    { name: "Lithuania", code: "LT" },
+    { name: "Luxembourg", code: "LU" },
+    { name: "Malta", code: "MT" },
+    { name: "Netherlands", code: "NL" },
+    { name: "Portugal", code: "PT" },
+    { name: "Slovakia", code: "SK" },
+    { name: "Slovenia", code: "SI" },
+    { name: "Spain", code: "ES" },
+  ];
+};
```

Comment:
This seems to be us duplicating logic about country names and codes. Let's centralize this in a constants file after we get all the PRs merged.